xml/installation-installation-rhel-provisioning_rhel.xml

```diff
@@ -189,24 +189,161 @@ gpgcheck=0</screen>
+     greater than 2.1.0.
+    </para>
+    <para>
+     The RPM packages needed for <literal>nova-compute</literal> must be
+     provided before <filename>site.yml</filename> execution is started. It is
+     expected that a &rhla; subscription will be used to provide these RPM
+     packages so that they get continuous updates.
+    </para>
+    <para>
+     The <literal>qemu-kvm</literal> packages can also be can be downloaded
+     from
+     <link xlink:href="http://vault.centos.org/7.3.1611/virt/x86_64/kvm-common/"/>.
+     The RPM packages below have been tested successfully:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       <filename>qemu-img-ev-2.6.0-28.el7.10.1.x86_64.rpm</filename>
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       <filename>qemu-kvm-common-ev-2.6.0-28.el7.10.1.x86_64.rpm</filename>
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       <filename>qemu-kvm-ev-2.6.0-28.el7.10.1.x86_64.rpm</filename>
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       <filename>qemu-kvm-tools-ev-2.6.0-28.el7.10.1.x86_64.rpm</filename>
+      </para>
+     </listitem>
+    </itemizedlist>
+   </step>
+   <step>
+    <para>
+     When using CentOS <literal>qemu-kvm</literal> RPM packages on &rhla;
+     compute nodes, SELinux policy updates are needed for
+     <literal>nova-compute</literal> to work. SELinux policy updates can be
+     enabled by setting a flag in the default values of
+     <literal>nova-compute</literal> plays. These flag values need to be set in
+     the &lcm; <emphasis role="bold">before</emphasis>
+     <literal>nova-compute</literal> plays are executed on &rhla; nodes.
+    </para>
+    <substeps>
+     <step>
+      <para>
+       On the &lcm;:
+      </para>
+<screen>
+cd ~/openstack/ardana/ansible/</screen>
+     </step>
+     <step>
+      <para>
+       Edit the <literal>nova-compute</literal> file with the flag defined in
+       ansible defaults.
+      </para>
+<screen>
+vi roles/NOV-CMP-KVM/defaults/main.yml
+					 </screen>
+     </step>
+     <step>
+      <para>
+       Set flag to <literal>true</literal>.
+      </para>
+<screen>
+nova_rhel_compute_apply_selinux_policy_updates: true
+					 </screen>
+     </step>
+     <step>
+      <para>
+       Save the file.
+      </para>
+     </step>
+     <step>
+      <para>
+       Commit the change in git.
+      </para>
+<screen>
+git add -A
+git commit --allow-empty -m "Enabled selinux policy updates for compute nodes"
+					 </screen>
+     </step>
+     <step>
+      <para>
+       Copy the change to scratch (work) area.
+      </para>
+<screen>
+ansible-playbook -i hosts/localhost ready-deployment.yml
+					 </screen>
+     </step>
+    </substeps>
+   </step>
+  </procedure>
+  <para>
+   Proceed with configuring &rhla; compute nodes.
+  </para>
+  <xi:include xpointer="element(/1/9)" href="installation-kvm_xpointer.xml"/>
```

Comment:
Dumb question, perhaps: this links to "Deploying the Cloud", not to a RHEL-specific section. Is that expected?

Reply:
Yes, it's expected. That section is in the HPE Installation Guide and was suggested by Arun Kant, who posted the original bug.